docs/tutorials/intro_tutorial.rst

```diff
@@ -384,213 +348,208 @@ those on the left edge, and the top to the bottom. This prevents some
+                
                 # Add the agent to a random grid cell
                 x = self.random.randrange(self.grid.width)
                 y = self.random.randrange(self.grid.height)
                 self.grid.place_agent(a, (x, y))
 
-Under the hood, each agent's position is stored in two ways: the agent
+Under the hood, each agent’s position is stored in two ways: the agent
 is contained in the grid in the cell it is currently in, and the agent
 has a ``pos`` variable with an (x, y) coordinate tuple. The
 ``place_agent`` method adds the coordinate to the agent automatically.
 
-Now we need to add to the agents' behaviors, letting them move around
+Now we need to add to the agents’ behaviors, letting them move around
 and only give money to other agents in the same cell.
 
-First let's handle movement, and have the agents move to a neighboring
-cell. The grid object provides a ``move_agent`` method, which like you'd
+First let’s handle movement, and have the agents move to a neighboring
+cell. The grid object provides a ``move_agent`` method, which like you’d
 imagine, moves an agent to a given cell. That still leaves us to get the
 possible neighboring cells to move to. There are a couple ways to do
 this. One is to use the current coordinates, and loop over all
 coordinates +/- 1 away from it. For example:
 
 .. code:: python
 
-    neighbors = []
-    x, y = self.pos
-    for dx in [-1, 0, 1]:
-        for dy in [-1, 0, 1]:
-            neighbors.append((x+dx, y+dy))
+   neighbors = []
+   x, y = self.pos
+   for dx in [-1, 0, 1]:
+       for dy in [-1, 0, 1]:
+           neighbors.append((x+dx, y+dy))
 
-But there's an even simpler way, using the grid's built-in
+But there’s an even simpler way, using the grid’s built-in
 ``get_neighborhood`` method, which returns all the neighbors of a given
-cell. This method can get two types of cell neighborhoods: Moore
-(including diagonals), and Von Neumann (only up/down/left/right). It
-also needs an argument as to whether to include the center cell itself
-as one of the neighbors.
+cell. This method can get two types of cell neighborhoods:
+`Moore <https://en.wikipedia.org/wiki/Moore_neighborhood>`__ (includes
+all 8 surrounding squares), and `Von
+Neumann <https://en.wikipedia.org/wiki/Von_Neumann_neighborhood>`__\ (only
+up/down/left/right). It also needs an argument as to whether to include
+the center cell itself as one of the neighbors.
 
-With that in mind, the agent's ``move`` method looks like this:
+With that in mind, the agent’s ``move`` method looks like this:
 
 .. code:: python
 
-    class MoneyAgent(Agent):
-       #...
-        def move(self):
-            possible_steps = self.model.grid.get_neighborhood(
-                self.pos,
-                moore=True,
-                include_center=False
-            )
-            new_position = self.random.choice(possible_steps)
-            self.model.grid.move_agent(self, new_position)
+   class MoneyAgent(Agent):
+      #...
```

Comment:
indentation: missing one space

Reply:
Sorry a little new to this, do you want me to change this? I can add the space

Reply:
would be great (otherwise copy and paste wont work)